docs/cli.adoc

```diff
@@ -48,6 +48,19 @@ This command has two modes:
 Incompatible with *--lsp-server*.
 --
 
+*--diagnostic-hyperlinks*=_when_::
+  Control whether the errors are using escape sequences to show errors as hyperlinks or not.
+  _when_ is one of the following:
++
+--
+- *auto* (default): shows errors as hyperlinks only if the error output is a terminal. On windows
+it behaves like 'never' because windows default terminals don't support escape sequences.
```

Comment:
Nit: One line per sentence:
```suggestion
- *auto* (default): shows errors as hyperlinks only if the error output is a terminal.
On windows it behaves like 'never' because windows default terminals don't support escape sequences.
```